api/database/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.3'
+
+services:
+  db:
+    build: ./db
+    container_name: postgres
+    ports:
+      - "5431:6666"
+    environment:
+      - POSTGRES_USER=SADashboard
+      - POSTGRES_PASSWORD=dashboard
+      - POSTGRES_DB=SAData
+    volumes:
+      - ./data:/data
```

Comment:
This should reference the actual path where the CSV lives rather than to a local copy not in version control, i.e. `/data` should be `../../src/data/`. And the reference to the CSV in the entrypoint COPY script should use the correct file name.